interfaces/Portalicious/src/app/pages/project-registration-personal-information/project-registration-personal-information.page.ts

```diff
@@ -62,20 +69,52 @@ export class ProjectRegistrationPersonalInformationPageComponent {
     ),
   );
 
+  project = injectQuery(this.projectApiService.getProject(this.projectId));
+
   attributeList = computed<DataListItem[]>(() => {
     const list: DataListItem[] = [];
     if (!this.projectAttributes.isSuccess() || !this.registration.isSuccess()) {
       return list;
     }
 
+    const genericAttributeNames =
+      this.getGenericAttributeNamesForPersonalInformation();
+    for (const attributeName of genericAttributeNames) {
+      const value = getValueForGenericAttribute(
+        this.registration.data()[attributeName],
+        attributeName,
+      );
+      const genericAttributeListItem = getGenericAttributeDataListItem(
+        attributeName,
+        value,
+      );
+      list.push(genericAttributeListItem);
+    }
     for (const attribute of this.projectAttributes.data()) {
       const value = this.registration.data()[attribute.name] as unknown;
-      const dataListItem = attributeToDataListItem(attribute, value);
+      const dataListItem = projectAttributeToDataListItem(attribute, value);
       if (dataListItem) {
         list.push(dataListItem);
       }
     }
-
     return list;
   });
+
+  private getGenericAttributeNamesForPersonalInformation(): GenericRegistrationAttributes[] {
```

Comment:
```suggestion
  private genericAttributeNamesForPersonalInformation = computed<GenericRegistrationAttributes[][]>(() => {
```

Watch out @RubenGeo @elwinschmitz ! Not following this pattern can lead to (hard to catch) bugs.

For additional context if needed: you are calling `getGenericAttributeNamesForPersonalInformation` from a `computed` variable (`attributeList`). That means that you would want `attributeList` to automatically update whenever `getGenericAttributeNamesForPersonalInformation` changes. If you do not make `getGenericAttributeNamesForPersonalInformation` a `computed` variable, then this will not happen, because angular will not know to react to changes within `getGenericAttributeNamesForPersonalInformation` itself (changes that could happen, for example, whenever the contents of `this.project.data()` changes).